### PR TITLE
Examples for iOS and Android

### PR DESCRIPTION
**Pull request by beast**
Finally tested most of things for example. Please review and merge.

### CODE REVIEW

**Comment by beast** [comment]
Hi, @jrichardlai . Is this repo still active? I'd love to contribute more but I haven't seen anything happened for a while.

**Comment by jrichardlai** [comment]
@beast sorry for the late response, I was on vacations and just came back. Wow thank you so much for this contribution, really appreciate it. I'll take a look tomorrow!

File: example/src/chat.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react'
+import {
+  AppRegistry,
+  StyleSheet,
+  Text,
+  TextInput,
+  TouchableHighlight,
+  View
+} from 'react-native';
+import ZendeskChat from '../node_modules/react-native-zendesk-chat'
```
Comment (jrichardlai) [inline example/src/chat.js]: I think this should be `import ZendeskChat from 'react-native-zendesk-chat'`

File: example/src/chat.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+export default class chatExample extends Component {
```
Comment (jrichardlai) [inline example/src/chat.js]: Can we use 2 spaces in the js file?

**Comment by jrichardlai** [comment]
The example seems to be working fine, I added some comments to the PR. Thanks

**Comment by jrichardlai** [comment]
Also can you use cocoapods https://github.com/zendesk/zendesk_sdk_ios for the example, thank you?

File: example/src/chat.js (hunk 2 above)
Comment (beast) [inline example/src/chat.js]: Go ahead. I am not religious in tab spacings :)

**Comment by beast** [comment]
Welcome back @jrichardlai I've been away for a while too. We are launching our own app soon and I am not sure when I will have time again for this. Feel free to cherry pick the PR. Thanks.

**Comment by beast** [comment]
@jrichardlai Hi there. I recently tried to implement a library myself that wraps another library. What I realize is that using cocoapods nicely with React library is really painful. https://github.com/beast/react-native-isDeviceRooted

I checked another library that wraps https://github.com/auth0/react-native-lock.  The instruction is even more complicated.

I just can't think of a good way to automate the example to work out of box without people having to read through the doc carefully. Do you have any ideas other than putting in more detailed instructions?

**Pull request closed without merging**